package.json

```diff
@@ -40,6 +40,7 @@
     "dotenv": "^14.2.0",
     "framer-motion": "^5.6.0",
     "http-status-codes": "^2.1.4",
+    "moment": "^2.29.1",
```

Comment:
We should avoid adding this library. It's pretty bloated and now deprecated. Consider a modern alternative (https://momentjs.com/docs/#/-project-status/) or just writing a helper function using the plain JS date API.